### PR TITLE
Staked farms tab

### PR DESCRIPTION
**Pull request by Oli8**
image

"Staked Farms" tab on the farms page to quickly get to those you deposited to.

### CODE REVIEW

File: src/Pages/Farms.js [hunk 1]
```diff
@@ -139,6 +143,7 @@ const Farms = (props) => {
               <Tab eventKey={FARM_TAB.ALL} title={FARM_TAB.ALL} />
               <Tab eventKey={FARM_TAB.YOU} title={FARM_TAB.YOU} />
               <Tab eventKey={FARM_TAB.CTEZ} title={FARM_TAB.CTEZ} />
+              <Tab eventKey={FARM_TAB.STAKED} title={FARM_TAB.STAKED} />
```
Comment (vijaypushkin) [inline src/Pages/Farms.js:146]: Can you make the tab scrollable in Mobile?

**Review by vijaypushkin — CHANGES_REQUESTED** [review]

File: src/Pages/Farms.js (hunk 1 above)
Comment (vijaypushkin) [inline src/Pages/Farms.js:146]: Can you make tabs scrollable in mobile view?

**Review by vijaypushkin — CHANGES_REQUESTED** [review]
Hey, Congrats on your first PR! 

Please resolve one review comment

File: src/Pages/Farms.js (hunk 1 above)
Comment (Oli8) [reply src/Pages/Farms.js:146]: Not sure what you mean, the tabs content or the tabs label list ?

Comment (berndoostrum) [reply src/Pages/Farms.js:146]: epic stuff @Oli8! We were thinking about adding something like that, but we need to find a way to differentiate 3 other tabs with "staked farms". Maybe we could add a switch instead like pancakeswap? 
image

Comment (Oli8) [reply src/Pages/Farms.js:146]: image

I got this now, you can horizontally scroll to the staked farm tab.
What do you think?

Comment (Oli8) [reply src/Pages/Farms.js:146]: > epic stuff @Oli8! We were thinking about adding something like that, but we need to find a way to differentiate 3 other tabs with "staked farms". Maybe we could add a switch instead like pancakeswap? image

Thanks ! Yeah maybe a switch would make more sense and would be better ux wise, I'll try something tomorrow.

Comment (Oli8) [reply src/Pages/Farms.js]: image
Implemented the switch

Comment (Oli8) [reply src/Pages/Farms.js]: Guys ? :)

**Comment by Oli8** [comment]
Up

File: src/Pages/Farms.js (hunk 1 above)
Comment (malviyaop) [reply src/Pages/Farms.js]: @vijaypushkin can you review now?

File: src/Pages/Farms.js [hunk 2]
```diff
@@ -111,9 +112,13 @@ const Farms = (props) => {
   );
 
   const farmsToRender = useMemo(() => {
-    const farmsInView = props.isActiveOpen
-      ? props.activeFarms.slice()
-      : props.inactiveFarms.slice();
+    let farmsInView = props.isActiveOpen ? props.activeFarms.slice() : props.inactiveFarms.slice();
+
+    if (props.isStakedOnlyOpen) {
```
Comment (vijaypushkin) [inline src/Pages/Farms.js:117]: Follow the same method as `filterBySearch` and `filterByTab`

**Review by vijaypushkin — CHANGES_REQUESTED** [review]

File: src/Pages/Farms.js (hunk 2 above)
Comment (Oli8) [reply src/Pages/Farms.js]: Done

**Review by vijaypushkin — APPROVED** [review]

**Pull request merged**